Library/Formula/llvm.rb

```diff
@@ -236,10 +230,20 @@ def install
   end
 
   def caveats
-    <<-EOS.undent
+    s = <<-EOS.undent
       LLVM executables are installed in #{opt_bin}.
       Extra tools are installed in #{opt_share}/llvm.
     EOS
+
+    if build.with? "libcxx"
+      s += <<-EOS.undent
+        To use the bundled libc++, please adjust your LDFLAGS as following:
+
+        LDFLAGS="-L#{opt_lib} -lc++abi"
```

Comment:
Well as @zhipeng-jia noted in the issue, the built libc++ doesn't add indirect symbols for libc++abi, so `-lc++abi` is still required, even with system libc++abi.